create-container.sh

```diff
@@ -5,15 +5,81 @@ set -e
 # Uncomment the following line to debug the script
 # set -x
 
+function print_help {
+    echo """
+    Usage: devenv [subcommand]
+
+A bash script to create and manage development environments using privileged LXC linux containers.
+
+Subcommands:
+    init - Generate the default configuration file. Create the .devenv
+
+
+More info: https://github.com/coopdevs/devenv
+    """
+}
+
+function init {
+    CONFIG="$1"
+    if [ -f "$CONFIG" ]; then
+      printf "The devenv config file already exists. Execute:\n\n$ cat .devenv\n"
+      exit 1
+    else
+      cat > "${CONFIG}" << EOF
+# File created with devenv init command
+
+NAME="<container name>"
+DISTRIBUTION="<SO distribution>"
+RELEASE="<SO release>"
+ARCH="<SO arch>"
+HOST="local.$NAME.coop"
```

Comment:
```suggestion
HOST="$NAME.local"
```

We've been using this other pattern since some time already, at least me. I prefer it this way because the '.local' tld doesn't exist and there are already [some protocols](https://en.wikipedia.org/wiki/Multicast_DNS#Protocol_overview) that understand that this is a LAN domain, at most.

Reply:
we do use that in other projects as well, I think